etna/loggers/wandb_logger.py

```diff
@@ -50,6 +47,7 @@ def __init__(
         table: bool = True,
         name_prefix: str = "",
         config: Optional[Union[Dict, str, None]] = None,
+        log_model: bool = False,
```

Comment:
Is this fix?

Reply:
As I've said about it on Daily: I had seen hardcoded `log_model` params in `def pl_logger`. This is bug that don't affect because of another bug)